bot.py

```diff
@@ -642,7 +645,7 @@ def check(r, u):
     output = await ctx.send(strings['nyoom_calc']['status']['calculating'])
     username = opted_in(user_id=user.id)
     if not username:
-        return await output.edit(content=output.content + strings['nyoom_calc']['status']['not_opted_in'])
+        return await output.edit(content=output.content + '\n' + strings['nyoom_calc']['status']['not_opted_in'])

```

Comment:
Hey err, as a quick thing, I'm going to reverse this to just await output.edit(strings['nyoom_calc']['status']['not_opted_in']) as that means the loading goes away, and the user doesn't need to know is *was* loaded, just when it *is* loading

Reply:
Actually, ignore that, didn't spot this was an except, thought it was the final return, 🙃